internal/journal/voljournal.go

```diff
@@ -401,7 +401,8 @@ func (conn *Connection) UndoReservation(ctx context.Context,
 
 	cj := conn.config
 	if volName != "" {
-		if len(volName) < 36 {
+		const minVolLen = 36
+		if len(volName) < minVolLen {
```

Comment:
This is part of commit `cleanup: address unused linters`. The description of the commit is completely unclear to me.

Better to identify where the `36` comes from. Just putting it in a variable and using it once is not much of an improvement.

Reply:
Oh Sorry, Due to continuous rebasing many changes are added to unrelated commits. I will restructure all the commits to a cleaner one.